notifications.bs

```diff
@@ -404,17 +404,32 @@ must be run.
  <li><p>Wait for any <a lt="fetch">fetches</a> to complete and <var>notification</var>'s
  <a>icon resource</a> and <a>sound resource</a> to be set (if any).
 
- <li>
-  <p>Replace <var>old</var> with <var>new</var>, in the same position, in the
+ <li><p>Replace <var>old</var> with <var>new</var>, in the same position, in the
   <a>list of notifications</a>.
 
-  <p>If the notification platform does not support replacement this requirement may be
-  addressed by running the <a>close steps</a> for <var>old</var> and then running the
-  <a>display steps</a> for <var>new</var>.
+ <li><p>If the <a>renotify preference flag</a> has been set, perform the
```

Comment:
s/perform x with/perform x for/ to remain consistent with the earlier invocation.